aptihramy-frontend/src/pages/TrackingChain.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+    <v-col>
+        <v-card class="pa-4">
+            <v-row>
+                <v-col>
+                    <v-card-title>Track: {{ personToDisplay[0].chef_prenom }} {{
+                        personToDisplay[0].chef_nom }}</v-card-title>
+                </v-col>
+
+                <v-col class="d-flex justify-end">
+                    <v-tooltip location="bottom">
+                        <template v-slot:activator="{ props }">
+                            <v-btn icon color="secondary" @click="resetZoom" large v-bind="props" class="mx-2">
+                                <v-icon>mdi mdi-restore</v-icon> <!-- Left Arrow -->
+                            </v-btn>
+                        </template>
+                        <span>Reset zoom</span>
+                    </v-tooltip>
+
+                    <v-tooltip location="bottom">
+                        <template v-slot:activator="{ props }">
+                            <v-btn icon color="secondary" @click="selectSomeElse" large v-bind="props" class="mx-2">
+                                <v-icon>mdi mdi-home</v-icon> <!-- Left Arrow -->
+                            </v-btn>
+                        </template>
+                        <span>Home</span>
+                    </v-tooltip>
+
+                    <v-tooltip location="bottom">
+                        <template v-slot:activator="{ props }">
+                            <v-btn icon color="secondary" @click="editPage" large v-bind="props" class="mx-2">
+                                <v-icon>mdi mdi-pencil</v-icon> <!-- Left Arrow -->
+                            </v-btn>
+                        </template>
```

Comment:
Would it make sense (I don't know how complicated it is) to create a TopBar component ?